dsr01:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

  PD_Gravity:
    type: m0609_controller/PD_Gravity
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6

    gains:
      joint1:
        pid: {p: 20.0, i: 100.0, d: 1.0}
      joint2:
        pid: {p: 20.0, i: 100.0, d: 1.0}
      joint3:
        pid: {p: 20.0, i: 100.0, d: 1.0}
      joint4:
        pid: {p: 20.0, i: 100.0, d: 1.0}
      joint5:
        pid: {p: 20.0, i: 100.0, d: 1.0}
      joint6:
        pid: {p: 20.0, i: 100.0, d: 1.0}


    root_link: base_0
    tip_link1: link6

    ctr_obj: 1

    ik_mode: 2

    clik_gain: {K_regulation: 30, K_tracking: 1.0}
#30
